code/modules/admin/verbs/adminfun.dm

```diff
@@ -60,11 +60,11 @@
 	if(!check_rights(R_ADMIN))
 		return
 
-	var/confirm = tgui_alert(usr, "Drop a brain?", "Confirm", list("Yes", "No","Cancel"))
+	var/confirm = tgui_alert(usr, "Drop a brain?", "Confirm", list("Yes", "No","Cancel")) || "Cancel"
 	if(confirm == "Cancel")
 		return
```

Comment:
```suggestion
	var/confirm = tgui_alert(usr, "Drop a brain?", "Confirm", list("Yes", "Cancel"))
	if(confirm != "Yes")
		return
```

We don't need both "No" and "Cancel" when they do the same thing, and also this is the cleanest possible way to explicitly check for a specific response